openapi-generator/4.3.1.wso2v1/pom.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
```

Comment:
The licence header should be 2020 right?

Reply:
Fixed